README.md

```diff
@@ -1,61 +1,93 @@
+- [Forms](packages/esm-patient-forms-app/README.md)
+- [Immunizations](packages/esm-patient-immunizations-app/README.md)
+- [Medications](packages/esm-patient-medications-app/README.md)
+- [Notes](packages/esm-patient-notes-app/README.md)
+- [Patient banner](packages/esm-patient-banner-app/README.md)
+- [Patient chart](packages/esm-patient-chart-app/README.md)
+- [Programs](packages/esm-patient-programs-app/README.md)
+- [Test results](packages/esm-patient-test-results-app/README.md)
+- [Vitals](packages/esm-patient-vitals-app/README.md)
 
-openmrs-esm-patient-chart is the patient chart frontend module and all its widgets. This creates a patient dashboard microfrontend for the OpenMRS SPA. It provides a simple dashboard with cards detailing the
-patient's information, such as vitals, demographic and relationships.
+In addition to these widgets, two other microfrontends exist that encapsulate cross-cutting concerns. These are:
+
+- [Common lib](packages/esm-patient-common-lib/README.md)
+- [Patient chart](packages/esm-patient-chart-app/README.md)
 
 ## Setup
 
-See the guidance in the [Developer Documentation](https://openmrs.github.io/openmrs-esm-core/#/getting_started/setup?id=developing-microfrontends-in-a-lerna-monorepo).
-This repository uses Yarn and Lerna.
+Check out the developer documentation [here](http://o3-dev.docs.openmrs.org).
+
+This monorepo uses [yarn](https://yarnpkg.com) and [lerna](https://github.com/lerna/lerna). 
 
-To start the dev server for a specific package, run
+To start a dev server for a specific microfrontend, run:
 
 ```bash
-yarn start --sources 'packages/esm-patient-<package>-app'
+yarn start --sources 'packages/esm-patient-<insert-package-name>-app'
 ```
 
-This will start a dev server for that package as well as `esm-patient-chart-app`.
+This command uses the `openmrs` tooling to fire up a dev server on the default port (`8081`) for the specified microfrontend in addition to `esm-patient-chart-app`.
 
-To start a dev server running all the packages, run
+To start a dev server running all the packages, run:
 
 ```bash
 yarn start-all
 ```
 
-Note that this is extremely resource-intensive.
+Note that this is very resource-intensive. 
+
+The recommended approach for working on multiple microfrontends is to run each one separately on a different port and then use [import map overrides](http://o3-dev.docs.openmrs.org/#/getting_started/setup?id=import-map-overrides). 
+
+For example, to work on the vitals and biometrics widgets simultaneously, you could run:
+
+```bash
+yarn start --sources 'packages/esm-patient-vitals-app' --port=8000
+
+yarn start --sources 'packages/esm-patient-biometrics-app' --port=9000
+```
+
+These commands would fire up the vitals app on `http://localhost:8001/openmrs-esm-patient-vitals-app` and the biometrics app on `http://localhost:9001/openmrs-esm-patient-biometrics-app`. You can use these import map URLs to override the import maps in your application's import map overrides panel.
```

Comment:
Are you intentionally adding `1` to the port? That is, `--port=8000` running at `localhost:8001` and `--port=9000` running at `localhost:9001`

Reply:
Yes. The [openmrs tooling](https://github.com/openmrs/openmrs-esm-core/blob/98a6c6bcc224e292fca2d1aa5e65a4ef1f7d55f4/packages/tooling/openmrs/src/utils/importmap.ts#L102) spins up an express server on the provided port. It also spins up a webpack dev server on a [different port](https://github.com/openmrs/openmrs-esm-core/blob/98a6c6bcc224e292fca2d1aa5e65a4ef1f7d55f4/packages/tooling/openmrs/src/utils/importmap.ts#L117) (derived from incrementing the provided port number by 1).

Reply:
Oh i see! Thanks for the clarification. 😊